tools/reviewer/webapp/src/app/dashboard/diff/diff-discussion/discussion.service.ts

```diff
@@ -36,13 +36,13 @@ export class DiscussionService {
       .length;
   }
 
-  // If new thread wasn't added or a thread wasn't deleted make links update.
+  // If new thread wasn't added or a thread wasn't deleted then make "links update".
```

Comment:
What is "links update"? Please explain in the comment.

Reply:
?????????????????????

Reply:
:upside_down_face:

Reply:
But there's already the explanation on the next line

Reply:
:upside_down_face:

Reply:
But there's already the explanation on the next line
https://i.imgur.com/FVVq76Q.png